### PR TITLE
Table for contributors section

### LINKED ISSUE #3713: Use definition table for contributors on Books Page
Just a small design request. In the editions #edition-details section, the details like id's, physical details... are styled with definition tables which produces a nicely structured view while the contributors section consists of list elements with lots of breaks which looks very cramped.

Just compare the sections in this example: [OL28775995M](https://openlibrary.org/books/OL28775995M/Der_Todeskanal#edition-details)

Style the contributors section with the same `<dt><dl><dd>` elements as the other sections.

Sorry that I can't make the change myself but I couldn't find the corresponding file and even than I'm not sure if I know how to change that.

### PR DESCRIPTION
**Pull request by Yashs911**
Closes #3713 

Table for Contributor Section

### Screenshot

ol work

### CODE REVIEW

File: openlibrary/templates/type/edition/view.html [hunk 1]
```diff
@@ -407,10 +407,10 @@ <h3 class="header">
           $if contributors:
             <div class="section">
               <h3 class="collapse">$_("Contributors")</h3>
-              <ul class="contributors">
-                $for c in contributors:
-                  $:display_contributor(c)
```
Comment (jdlrobson) [inline openlibrary/templates/type/edition/view.html:412]: Nothing in this template appears to be using `display_contributor` now so the macro should be removed.

https://github.com/internetarchive/openlibrary/pull/4373/files#diff-93ec72c37de495b619f24641f2b9defc4ea1205bb5450bb6baa33498831721efR126

**Review by jdlrobson — CHANGES_REQUESTED** [review]

File: openlibrary/templates/type/edition/view.html (hunk 1 above)
Comment (Yashs911) [reply openlibrary/templates/type/edition/view.html:412]: @jdlrobson done

**Comment by jdlrobson** [comment]
How can I test this locally @yashs911  ? Can you remind me how to import articles? I've forgotten - it's been a while :)

**Comment by Yashs911** [comment]
@jdlrobson 
docker-compose exec web bash
./scripts/copydocs.py /books/OL28775995M

**Review by jdlrobson — APPROVED** [review]

**Comment by jdlrobson** [comment]
Thanks for that. Looks like I've got a few problems with my install however so I will need to sort that out before merging.

If someone can verify the fix in the mean time I think this is good to go.

**Comment by jdlrobson** [comment]
This looks so much better. Thanks for working on it. While testing I noticed the headings for `The Physical Object` and `ID Numbers`  are squashed compared to the `Contributors` heading . Is that something you would be interested in looking at in a follow up ?

Screen Shot 2021-01-07 at 8 43 39 PM

**Comment by Yashs911** [comment]
@jdlrobson Okay, I will create a PR soon

**Comment by Yashs911** [comment]
@jdlrobson `The Physical Object` and `ID Numbers` are squashed compared to the Contributors heading:
It is due to `class="list-header collapse"`

Should I remove the float or change the class to `class="collapse"`

image

**Comment by jdlrobson** [comment]
Semantically everything is correct here. I think the issue here is the rule `div.editionAbout h3` clashing with `.list-header`.  I would suggest adding a new class `edition-header` to all elements that match `div.editionAbout h3` and simplifying that selector to `.edition-header` to remove the clash.

**Comment by Yashs911** [comment]
@jdlrobson I just had one doubt do I need to change `div.editionAbout h3` to `div.editionAbout.edition-header`?

**Pull request merged**